keras_core/layers/preprocessing/random_crop.py

```diff
@@ -3,10 +3,14 @@
 from keras_core import backend
 from keras_core.api_export import keras_core_export
 from keras_core.layers.layer import Layer
-from keras_core.utils import backend_utils
+from keras_core.utils import backend_utils, image_utils
 from keras_core.utils.module_utils import tensorflow as tf
 
 
+H_AXIS = -3
+W_AXIS = -2
```

Comment:
Depends if we're using channels_first or channels_last. Long term if we're backend agnostic we should support both (and there should be a `data_format` argument in `__init__`)